abi/serializer.go

```diff
@@ -187,7 +191,7 @@ func (s *serializer) serializeInputVariadicValues(partsHolder *partsHolder, valu
 	return nil
 }
 
-func (s *serializer) serializeDirectlyEncodableValue(partsHolder *partsHolder, value any) error {
+func (s *serializer) serializeSingleValue(partsHolder *partsHolder, value singleValue) error {
```

Comment:
`single value` is the correct, official name: https://docs.multiversx.com/developers/data/multi-values.